mailbox/tools/quota-recompute/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-data-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-data-jmap</artifactId>
+        </dependency>
```

Comment:
No we MUST not add this dependency here.

We MUST provide a way to inject our components without requiring a hard dependency.

We still do not obey to the word to the othogonal architeture and still need to further polish the implementation here to avoid uneeded dependencies before I can approuve this work.

Reply:
fixed